integ-test-promote.pipeline.yml

```diff
@@ -285,6 +325,83 @@ stages:
               test -f test-results/matlab/results.pdf
             displayName: Verify PDF report was created
             condition: eq(variables['Agent.OS'], 'Linux')
+            
+      - job: test_run_build
```

Comment:
What's going on here? Isn't this already in master?

Reply:
There was an issue while pulling the latest changes (related to run-build pushed by David) from MathWorks repo. It seems to be sorted now.